pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -166,3 +167,39 @@ func checkIfRunningAsNormalUser() error {
 	logging.Debug("Ran as administrator")
 	return fmt.Errorf("crc should be ran as a normal user")
 }
+
+func removeDnsServerAddress() error {
+	resetDnsForDefaultSwitchCommand := `Set-DnsClientServerAddress -InterfaceAlias ("vEthernet (crc)") -ResetServerAddresses`
+	if exist, defaultSwitch := winnet.GetDefaultSwitchName(); exist {
+		resetDnsForDefaultSwitchCommand = fmt.Sprintf(`Set-DnsClientServerAddress -InterfaceAlias ("vEthernet (%s)","vEthernet (crc)") -ResetServerAddresses`, defaultSwitch)
```

Comment:
why not insert the string for the default, by appending...

or collect the options and return the `-InterfaceAlias ("vEthernet (crc)", ...` accordingly.

Reply:
`resetDnsForDefaultSwitchCommand` is not the right name when only `crc` is used.

Rather call this `resetDnsForVirtualSwitchCommand`